test/index.js

```diff
@@ -30,4 +30,34 @@ describe('Pino Logger', () => {
     expect(seneca.log).to.exist()
     done()
   })
+
```

Comment:
This test will throw an exception if you remove the try catch.  It's only there to allow me to commit the files.

    cb is not a function